tests/xharness/Logging/CaptureLog.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.IO;
+
+namespace Xharness.Logging {
+
+	// A log that captures data written to a separate file between two moments in time
+	// (between StartCapture and StopCapture).
+	public class CaptureLog : Log
+	{
+		public string CapturePath { get; private set; }
```

Comment:
Can be `private readonly` field

Reply:
Would not set it private, we might use it, but we can remove the property and make it readonly.

Reply:
I would make it public when we need to use it? Maybe it's not going to happen..